sonic-pcied/scripts/pcied

```diff
@@ -57,17 +57,12 @@ class DaemonPcied(DaemonBase):
     def check_pcie_devices(self):
         try:
             platform_path, _ = device_info.get_paths_to_platform_and_hwsku_dirs()
-            platform_plugins_path = os.path.join(platform_path, "plugins")
-            sys.path.append(os.path.abspath(platform_plugins_path))
-            from pcieutil import PcieUtil
+            sys.path.append(os.path.abspath(platform_path))
```

Comment:
Why do we need to append `platform_path` to `sys.path`? We're not loading Python modules from here, correct? Only the pcie.yaml file resides there, right?

Reply:
I will change it.